src/resource.js

```diff
@@ -0,0 +1,23 @@
+const apiKey = 'iiT5pz1slpmXaLAWU1cygqoLpwoAeAu3';
+
+export const findWeather = async (locId) => {
+  const currentCondRes = 'https://dataservice.accuweather.com/currentconditions/v1/';
+
+  const queryParam = `${locId}?apikey=${apiKey}`;
+
+  const response = await fetch(currentCondRes + queryParam);
```

Comment:
- [ ] I have not been able to run any weather request for a city yet. Here is the error messages I get in the console.
![Screenshot from 2021-08-29 02-19-25](https://user-images.githubusercontent.com/65501535/131233400-bda64480-aaf8-4191-8065-9098bca1fa83.png)